fpm/build-deb

```diff
@@ -56,4 +54,7 @@ fpm -t deb \
     --description "Soundnode App is an Open-Source project to support Soundcloud for desktop Mac, Windows, and Linux." \
     --after-install "fpm/create-symlink" \
     --after-remove "fpm/remove-symlink" \
-    --deb-no-default-config-files
+    --deb-no-default-config-files \
+    "$DIST_DIR/.=/usr/share/soundnode/" \
+    "app/soundnode.png=/usr/share/pixmaps/" \
```

Comment:
@jakejarrett True. I changed that in https://github.com/Soundnode/soundnode-app/pull/777/commits/56e013fd242fc4be0dc504a385e9614f078f7fa1